### DISCUSSION TITLE
Deploying dashboard with additional YAML?

### DISCUSSION THREAD

**Discussion by MayTheForkBeWithU** [kubernetes/dashboard#9385] [Q&A]
To use bearer token authentication for the dashboard we need to have additional yaml files like clusterrole, service accounts, roles from my understanding.

Is there a way to do a single deployment with the additional yaml files?

**Comment by alecharp** [comment]
in the helm values, there is the `extras.manifests` key. I use this to create the  `ServiceAccount`, `ClusterRoleBinding` and `Secret`.
I don't know if that answer your question but that permitted me not to have an additional `kubectl apply -f [def].yaml` command to type.
